parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -23,6 +23,13 @@
 # KUBECONFIG_KEY ETCD_SERVER_CERTIFICATE ETCD_SERVER_PRIVATE_KEY ETCD_CLIENT_CERTIFICATE ETCD_CLIENT_PRIVATE_KEY
 # ETCD_PEER_CERTIFICATES ETCD_PEER_PRIVATE_KEYS ADMINUSER MASTER_INDEX
 
+# Capture Interesting Network Stuffs during provision
+packetCaptureProvision() {
+    tcpdump -G 600 -W 1 -n -vv -w /var/log/azure/dnsdump.pcap -Z root -i eth0 udp port 53 > /dev/null 2>&1 &
+}
+
+packetCaptureProvision
```

Comment:
Not sure how it will impact on the performance side, should it be put behind some debug flag?

Reply:
+1, does this significantly increase deployment duration?

Reply:
This will not have an observable impact on deployment time (unless there is a DNS storm, in which case we might want that data :))